test/functional/snaps.test.ts

```diff
@@ -2,36 +2,39 @@ import child_process from 'child_process';
 import { expect } from 'chai';
 import fs from 'fs';
 import util from 'util';
+import { normalize } from '../helpers.test';
 
 const exec = util.promisify(child_process.exec);
 
 // FIXME: assertions and done()
 describe('snap test', () => {
   const root = process.cwd();
 
-  it('should report OK for test without errors', () => {
+  it('should report OK for test without errors', async () => {
     return exec(
-      `node ${root}/dist/src/snapshot.js \\
```

Comment:
Windows cmd does not support escaping newlines like this, so we need to concat them all together in a single line.